man/znc.1

```diff
@@ -66,7 +67,7 @@ saves everything.
 .BR \-c ", " \-\-makeconf
 Interactively create a new configuration.
 .TP
-.BR \-s ", " \-\-makepass
+.BI \-s " [PASSWORD]" "\fR,\fP \-\-makepass" =[PASSWORD]
```

Comment:
It should say that it's only for automation. Because when the unsuspecting user is going to see `<Pass password>...</Pass>` without any explanation, what will they do?

Reply:
Done